docs/ui-coverage/core-concepts/views.mdx

```diff
@@ -9,30 +9,67 @@ sidebar_position: 100
 
 <UICovAddon />
 
-UI Coverage organizes all unique URLs across snapshots from end-to-end tests and mounted components from component tests into "Views". A View represents a distinct page or state in your application.
+UI Coverage organizes unique URLs from snapshots in end-to-end tests and mounted components in component tests into Views. Each View represents a distinct page or state in your application, allowing you to track coverage across different parts of your application.
 
-## View Creation & Grouping
+## How views are created
```

Comment:
We need to decide if "View" is a proper noun or not. It's inconsistent throughout